resources/eventing/charts/publisher-proxy/templates/_helpers.tpl

```diff
@@ -58,14 +58,6 @@ Selector labels for peerauthentication
 app.kubernetes.io/name: {{ include "publisher-proxy.fullname" . }}
```

Comment:
Can you please remove `publisher-proxy.peerauth.selectorLabels` as it is redundant now?

Reply:
I can't. It is still used here: https://github.com/kyma-project/kyma/blob/e386863bb4cd486cd323df6c6d01af10c4a87537/resources/eventing/charts/publisher-proxy/templates/peerauthentication.yaml#L9

Reply:
I can't. It is still used here: https://github.com/kyma-project/kyma/blob/e386863bb4cd486cd323df6c6d01af10c4a87537/resources/eventing/charts/publisher-proxy/templates/peerauthentication.yaml#L9

Reply:
I didn't want to do a bigger clean-up. Just remove duplicated keys to make the output YAML template parseable.